onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -6014,10 +6014,93 @@ def aten_var(self: TensorType, unbiased: bool = True) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_var_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType, TensorType]:
+@torch_op("aten::var_mean", trace_only=True)
+def aten_var_mean(self: TReal, unbiased: bool = True) -> tuple[TReal, TReal]:
     """var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)"""
 
-    raise NotImplementedError()
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean(self, correction, keepdim=False)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_dim(
+    self: TReal, dim: int, unbiased: bool = True, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    """var_mean.dim(Tensor self, int[1]? dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)"""
+
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean_dim(self, dim, correction, keepdim)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_correction(
+    self: TReal, dim: int = None, correction: int = 1, keepdim: bool = False
```

Comment:
```suggestion
    self: TReal, dim: Optional[int] = None, correction: Optional[int] = None, keepdim: bool = False
```

Is correction float or int, since it is "Scalar" below?

Reply:
https://pytorch.org/docs/master/generated/torch.var_mean.html?highlight=var_mean#torch.var_mean
Pytorch document say the correction is int.
I tried to set it to float, but FullGraph mode will be failed due to:
```
if correction != 0.0
```
Because the input value is int(2), not float(2.0), so the type mismatched.
So I would like to keep it to int.